src/app/userDashboard/tournament/CountdownComponent.tsx

```diff
@@ -1,19 +1,16 @@
 'use client';
-import React from 'react';
-import styles from '../../../styles/CountdownComponent.module.scss';
+const CountdownComponent = (dateAndTime: string, roomUuid: string, setVisibleRooms: React.Dispatch<React.SetStateAction<string[]>>) => {
```

Comment:
is this a component or just a function can't we understand a difference between a component and a function